apps/ios/react-native.config.js

```diff
@@ -18,7 +18,7 @@
 module.exports = {
   project: {
     ios: {
-      project: "apps/ios/src/FluentUITester.xcworkspace"
```

Comment:
Out of curiosity, what happened to cause it to need to be a relative path now?

Reply:
I don't think it's ever been relative? In `react-native-test-app`, we've always used it like this. I actually got an error here, which is why I had to change it in the first place.